testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java

```diff
@@ -256,6 +256,7 @@ public abstract class LayersTestBase {
             "org.wildfly.extension.metrics",
             // Extension not included in the default config
             "org.wildfly.extension.mvc-krazo",
+            "org.wildfly.extension.security.manager",
```

Comment:
I think this also needs to be in NO_LAYER_WILDFLY_PREVIEW to fix the LayersTestCase failure.